charts/flyte-core/values.yaml

```diff
@@ -944,7 +954,7 @@ configmap:
   datacatalogServer:
     datacatalog:
       storage-prefix: metadata/datacatalog
-      metrics-scope: datacatalog
+      metrics-scope: flyte
```

Comment:
Changed this to `flyte` otherwise the metrics look like `datacatalog:datacatalog:....`

With this change it looks like other metrics `flyte:datacatalog:...`